README.md

```diff
@@ -105,6 +106,12 @@ The following diagram can be updated using the [Diagrams.net Integration Intelli
 
 ![DB schema](diagrams/db-schema.svg)
 
+## UML class Diagram for Models
+The following diagram can be updated by editing the corresponding drawio file in diagrams and then exporting the drawing
+to png.
+
+![UML class Diagram for Models](diagrams/ModelUML.drawio.png)
```

Comment:
You should only need one file, not two - no exporting necessary. 
Collaborators updating the diagrams shouldn't have to export anything.

**Can you mimic how I have drawio implemented for the DB schema?**